### ISSUE TITLE
libc issue on Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by AsLY4** [Wilfred/difftastic#446]
Hello,

Last version of difftastic (0.39) introduce a dependancy on libc6 (>= 2.32)
```
difft -V                                                                                                                                                      
./difft: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./difft)
./difft: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./difft)
./difft: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./difft)
```

But on Ubuntu 20.04 LTS there is only libc6 2.31
```
apt-cache policy libc6
  Installed: 2.31-0ubuntu9.9
  Candidate: 2.31-0ubuntu9.9
```

Previous version of Difftastic works:
```
difft --version
Difftastic 0.38.0
```

Regards,

**Comment by tmysik** [comment]
The same for me, I am on RHEL:

```
$ difft --version
difft: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by difft)
difft: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by difft)
difft: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by difft)
```

**Comment by Wilfred** [comment]
Looks like ubuntu-latest on GitHub has changed to 22.04: https://github.blog/changelog/2022-11-09-github-actions-ubuntu-latest-workflows-will-use-ubuntu-22-04/

I'll change it back to 20.04 for the next release.

**Comment by Wilfred** [comment]
I've released 0.40.
